src/main/java/io/github/thebusybiscuit/slimefun4/utils/PatternUtils.java

```diff
@@ -14,7 +16,8 @@
  */
 public final class PatternUtils {
 
-    private PatternUtils() {}
+    private PatternUtils() {
+    }
```

Comment:
Either one-line or with a comment inside. But not this please :/

```suggestion
    private PatternUtils() {}
```

Reply:
We really have to get some sort of ruleset for intellij, the autoformat on commit just leads to so many unnecessarily rejected reviews...